R/pkg/R/generics.R

```diff
@@ -689,67 +689,67 @@ setGeneric("randomSplit", function(x, weights, seed) { standardGeneric("randomSp
 
 ###################### Column Methods ##########################
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("asc", function(x) { standardGeneric("asc") })
 
-#' @rdname column
+#' @rdname between
 #' @export
 setGeneric("between", function(x, bounds) { standardGeneric("between") })
 
-#' @rdname column
+#' @rdname cast
 #' @export
 setGeneric("cast", function(x, dataType) { standardGeneric("cast") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("contains", function(x, ...) { standardGeneric("contains") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("desc", function(x) { standardGeneric("desc") })
 
-#' @rdname column
+#' @rdname endsWith
 #' @export
 setGeneric("endsWith", function(x, suffix) { standardGeneric("endsWith") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("getField", function(x, ...) { standardGeneric("getField") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
 
-#' @rdname column
+#' @rdname column functions
 #' @export
 setGeneric("isNull", function(x) { standardGeneric("isNull") })
 
-#' @rdname column
+#' @rdname column functions
```

Comment:
They were on one page, but "column" is the wrong rd to put them - there's already the class `Column` and function `column`. (there's also function `columns` but that's a different page) - so the intent is to put these in their separate pages as `column functions`)

it looked like this previously:
![image](https://cloud.githubusercontent.com/assets/8969467/16219046/32d18c72-3738-11e6-88e6-1b8765b79770.png)

This is still buggy - investigating.